build/NermNermNerm.Stardew.LocalizeFromSource.targets

```diff
@@ -2,9 +2,9 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <UsingTask TaskName="Exec" AssemblyFile="$(MSBuildToolsPath)\Microsoft.Build.Tasks.Core.dll" />
   <Target Name="BuildI18nFolder" BeforeTargets="AfterBuild">
```

Comment:
On linux, would it work to simply chop off the '.exe' off of the path?  (Though I can't say with any conviction whether it's better than what you've got here.)

Reply:
In theory it would, yes. I just went with this since I figured it should be cross platform without needing to look into doing any logic there